lib/kitchen/verifier/pester.rb

```diff
@@ -413,6 +562,14 @@ def absolute_test_folder
         integration_path
```

Comment:
no need for a temp variable here.  Just emit the line above

Reply:
Do you mean:
```ruby
      def absolute_test_folder
        path = (Pathname.new config[:test_folder]).realpath
        integration_path = File.join(path, "integration")
        Dir.exist?(integration_path) ? integration_path : path
      end
```